tools/src/scie_pants/install_pants.py

```diff
@@ -67,8 +67,59 @@ def pip_install(*args: str) -> None:
     #
     # Also, we don't advance setuptools past 58 which drops support for the `setup` kwarg `use_2to3`
     # which Pants 1.x sdist dependencies (pystache) use.
-    pip_install("-U", "pip==22.3.1", "setuptools<58", "wheel")
-    pip_install("--progress-bar", "off", *pants_requirements)
+    venv_pip_install(venv_dir, "-U", "pip==22.3.1", "setuptools<58", "wheel", find_links=find_links)
+    venv_pip_install(venv_dir, "--progress-bar", "off", *pants_requirements, find_links=find_links)
+
+
+def install_pants_from_pex(
+    venv_dir: Path,
+    prompt: str,
+    version: Version,
+    ptex: Ptex,
+    extra_requirements: Iterable[str],
+    find_links: str | None,
+) -> None:
+    """Installs Pants into the venv using the platform-specific pre-built PEX."""
+    uname = os.uname()
+    major, minor = sys.version_info[:2]
+    pex_name = f"pants.cp{major}{minor}-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
+    with tempfile.NamedTemporaryFile(suffix=".pex") as pants_pex:
+        pex_url = (
+            f"https://github.com/pantsbuild/pants/releases/download/release_{version}/{pex_name}"
+        )
+        try:
+            ptex.fetch_to_fp(pex_url, pants_pex.file)
+        except subprocess.CalledProcessError as e:
+            fatal(
+                f"Wasn't able to fetch the Pants PEX at {pex_url}.\n\n"
+                + "Check to see if the URL is reachable (i.e. GitHub isn't down) and if the asset exists."
```

Comment:
It'd be good to suggest how one can check the asset exists:

```suggestion
                + f"Check to see if the URL is reachable (i.e. GitHub isn't down) and if {pex_name} asset exists within the {version} release."
```

And potentially even linking to the actual release page for a human to check.

Another route for failure is that the Pants release doesn't exist (e.g. someone tries `PANTS_VERSION=2.1234.5678`)? Or is that detected earlier?